### DISCUSSION TITLE
MySQL syntax error

### DISCUSSION THREAD

**Discussion by philharmonie** [requarks/wiki#3794] [Error / Bug Report] [answered]
I did a fresh install of Wiki.JS. When I try to upload an image, I receive the following error:

```[GraphQL error]: Message: with recursive `ancestors` as (select `id`, `name`, `slug`, `parentId` from `assetFolders` where `id` = 0 union select `a`.`id`, `a`.`name`, `a`.`slug`, `a`.`parentId` from `assetFolders` as `a` inner join `ancestors` on `ancestors`.`parentId` = `a`.`id`) select * from `ancestors` - You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'recursive `ancestors` as (select `id`, `name`, `slug`, `parentId` from `assetFol' at line 1, Location: [object Object], Path: assets,folders```

**Comment by NGPixel** [comment, marked as answer]
You're not using a supported MySQL version... Check the requirements.
